docs/app/Examples/modules/Dropdown/Content/AsyncOptions.js

```diff
@@ -25,22 +28,31 @@ export default class DropdownAsyncOptions extends Component {
   }
 
   selectRandom = () => this.setState({ value: _.sample(this.state.options).value })
+  toggleSearch = (e) => this.setState({ search: e.target.checked })
 
   render() {
-    const { options, isFetching, value } = this.state
-    const classes = cx('search selection multiple', { 'disabled loading': isFetching })
+    const { options, isFetching, search, value } = this.state
 
     return (
       <div>
         <Button onClick={this.fetchOptions}>Fetch</Button>
         <Button onClick={this.selectRandom} disabled={_.isEmpty(options)}>Random</Button>
         <Dropdown
-          defaultText='Add Users'
-          className={classes}
           options={options}
           value={value}
+          placeholder='Add Users'
+          search={search}
+          selection
           onChange={this.handleChange}
+          disabled={isFetching}
+          loading={isFetching}
         />
+        <p>
+          <label>
+            <input type='checkbox' value={search} onChange={this.toggleSearch} />
+            {' '}Search
+          </label>
+        </p>
```

Comment:
The docs are getting an overhaul to better demonstrate the v1.0 component APIs.  Take this example with a grain of salt.  It is here only to show that the Dropdown works as expected.